### ISSUE TITLE
Favorites tumangaonline.com in red no found

### ISSUE THREAD

**Issue by VonKairak** [riderkick/FMD#744]
since a week ago can not download in favorites from the site tumanga online, I thought it was a temporary problem or network, but not to run on other networks and it is not solved, please check the problem to continue enjoying such a good program.

regards

**Comment by VonKairak** [comment]
![image](https://user-images.githubusercontent.com/32087410/30914339-c12da85e-a369-11e7-92b0-3ce043243302.png)
my Version : 0.9.126.0

**Comment by OldMartin** [comment]
Same problem

**Comment by kmvi** [comment]
fixed in 133 release

**Comment by juniornff** [comment]
Me pasa casi lo mismo.
Al principio todo va bien. Pero cuando esta revisando si hay nuevos capíitulos, cuando revisa cierta cantidad de mangas, empiezan a ponerse en rojo y no me permite descargar. Despues de un tiempo me permite descargar los capitulos.
![1](https://user-images.githubusercontent.com/33964836/53813601-5ea17680-3f34-11e9-9b57-9e8d36929e70.PNG)

Cuando vi eso copie el link del manga y lo use en el navegador, el cual me dice que he hecho muchas peticiones.
![2](https://user-images.githubusercontent.com/33964836/53813614-66f9b180-3f34-11e9-86a9-5efb38f2b566.PNG)

**Comment by juniornff** [comment]
It happens to me almost the same.
At first everything is fine. But when you are checking if there are new chapters, when you check a certain amount of sleeves, they start to turn red and it does not allow me to download. After a while it allows me to download the chapters.
![1](https://user-images.githubusercontent.com/33964836/53813601-5ea17680-3f34-11e9-9b57-9e8d36929e70.PNG)

When I saw that, I copied the manga link and used it in the browser, which tells me that I have made many requests.
![2](https://user-images.githubusercontent.com/33964836/53813614-66f9b180-3f34-11e9-86a9-5efb38f2b566.PNG)
